.github/workflows/ci.yml

```
@@ -0,0 +1,109 @@
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v2
+
+      - name: Build and push API Server image
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: shortlink-api
+          IMAGE_TAG: ${{ github.sha }}
+        working-directory: ./BE/api-server
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:17: Double quote to prevent globbing and word splitting

(shellcheck)